File: 2540-minimum-common-value/2540-minimum-common-value.py
class Solution:
    def getCommon(self, nums1: List[int], nums2: List[int]) -> int:
        l=0
        r=0
        l1=len(nums1)
        l2=len(nums2)
        while (l<l1 and r<l2):
            if nums1[l]==nums2[r]:
                return nums1[l]
            elif nums1[l]<nums2[r]:
                l+=1
            else:
                r+=1
        return -1
            

File: 2540-minimum-common-value/2540-minimum-common-value.java
class Solution {
    public int getCommon(int[] nums1, int[] nums2) {
        int l=0,r=0;
        int l1=nums1.length;
        int l2=nums2.length;
        while (l<l1 && r<l2){
            if(nums1[l]==nums2[r]){
                return nums1[l];
            }
            else if(nums1[l]<nums2[r]){
                l++;
            }
            else{
                r++;
            }
        }
        return -1;
    }
}